CHANGELOG.md

```diff
@@ -38,6 +38,7 @@
 - \#2845 - Misleading error feedback in app creation modal on port conflict
 - \#2812 - Cannot change configuration of Marathon app after deployment
 - \#2821 - Tooltip content update issue
+- \#2972 - Add more inline task information
```

Comment:
yeah, but we released that bug in 0.14.1, so I think it should read

## 0.14.2
### Fixed
 - \#2972 - always show endpoints in tasks view